perceval/backends/core/confluence.py

```diff
@@ -279,6 +291,12 @@ def setup_cmd_parser():
         parser.parser.add_argument('url',
                                    help="URL of the Confluence server")
 
+        # Optional arguments
```

Comment:
this code is not needed if the ancestor info is included by default in the data returned by the backend